app/views/shared/_invite_users_modal.html.erb

```diff
@@ -74,29 +74,40 @@ invite_to_team = type.in?(%w(invite_to_team invite_to_team_with_role))
           <div class="user-selector">
             <select class="emails-input" multiple data-role="tags-input" name="emails[]">
             </select>
-            <br />
-            <label><%= t('invite_users.input_subtitle') %></label>
+            <h4><%= t('invite_users.input_subtitle') %></h4>
           </div>
 
           <% if current_user && type.in?(['invite_with_team_selector', 'invite_with_team_selector_and_role']) %>
             <% # Only allow inviting to teams where user is admin %>
-            <% teams = current_user.teams
-                                   .joins(:user_teams)
-                                   .where('user_teams.role': UserTeam.roles[:admin])
-                                   .select { |team|  can_invite_team_users?(team) } %>
-            <% if teams.any? %>
-              <div class="team-selector">
-                <div class="heading">
-                  <input type="checkbox" data-role="team-selector-checkbox" />
-                  <span><%= t('invite_users.invite_to_team_heading') %></span>
+            <select class="team-ids-input" name="team_ids[]" style="display: none" multiple>
+            </select>
+            <div id="team-selector" class="team-selector">
```

Comment:
the same ID as class name?

Reply:
Is this an issue? Classes are usually used for CSS, IDs for JS targeting.